python/nav/models/api.py

```diff
@@ -45,13 +45,11 @@ class APIToken(models.Model):
     comment = models.TextField(null=True, blank=True)
     revoked = models.BooleanField(default=False)
     last_used = models.DateTimeField(null=True)
-    endpoints = hstore.DictionaryField(null=True, blank=True)
+    endpoints = HStoreField(null=True, blank=True, default={})
```

Comment:
Setting the default value is probably a good idea - but how does this behave under 1.7 and 1.8 when a pre-existing database contains NULL values in these fields?